doc/contributing/writing-tests.md

```diff
@@ -139,6 +139,28 @@ request. Interesting things to notice:
   exit gracefully. Remember that for a test to succeed, it must exit with a
   status code of 0.
 
+## Running tests
+
+All tests can be executed by running the `jstest` target:
```

Comment:
It won't run all the tests, as its name implies it's only a subset of them.
```suggestion
Running the `jstest` target will execute all the tests for the `node` binary:
```

Reply:
I think there is still a catch in this...that `internet` tests aren't run with `make jstest` 😂

Reply:
```suggestion
Running the `jstest` target will execute all the tests for the `node` binary
except those which require internet access:
```